LabExT/Experiments/StandardExperiment.py

```diff
@@ -457,6 +462,11 @@ def load_measurement_dataset(self, meas_dict, file_path, force_gui_update=True):
         # add file path to dictionary
         meas_dict["file_path_known"] = file_path
 
+        if self._meas_control_settings.finished_meas_limited:
+            while len(self.measurements_hashes) >= self._meas_control_settings.max_finished_meas:
+                self.measurements_hashes.pop(0)
+                self.measurements.pop(0)
```

Comment:
I recommend improving the logic here a bit:

Search for the oldest measurement using the `timestamp_known` key. Then, find the hash of that measurement and delete that hash from both of the lists you modify here. I think this would make most sense here - and matches most what you wrote in the tooltip for the corresponding setting.

Reply:
I have changed this now according to your suggestions. Before it was FIFO (first in first out) based on when it was added to the measurement list `self.measurements`, whereas now it's FIFO based on the timestamp of each loaded measurement. This means that if you add/load an older measurement it will be the first one to be removed again if the limit is reached and even if it was the last measurement that was added.